optimade/models/structures.py

```diff
@@ -327,9 +333,13 @@ class StructureResourceAttributes(EntryResourceAttributes):
   - For a wire along the direction specified by the third lattice vector: :val:`[0, 0, 1]`
   - For a 2D surface/slab, periodic on the plane defined by the first and third lattice vectors: :val:`[1, 0, 1]`
   - For a bulk 3D system: :val:`[1, 1, 1]`""",
+        min_items=3,
+        max_items=3,
     )
 
-    lattice_vectors: Optional[List[conlist(len_eq=3)]] = Field(
+    lattice_vectors: Optional[
```

Comment:
it might be prudent to define a stand-in for a 3D vector:
`Vector3D = Tuple[Union[float,None], Union[float,None], Union[float,None]]`

Then the same comment as before, making this a `Tuple` of 3 `Vector3D`s seems appropriate since the description says this is three lattice vectors.